### DISCUSSION TITLE
Dark Mode

### DISCUSSION THREAD

**Discussion by jhart003** [ExperienceLovelace/ha-floorplan#158] [Q&A] [answered]
How do you deal with the browser automatically switching to dark mode?  I have a working floor plan that has a white background. Unfortunately if I view on my iPad or iPhone after 5PM I can’t see much because my devices automatically switch into dark mode. 

It looks like the picture card route allows you to specify a background when the browser is in dark mode. Is there a similar feature for ha-floor plan?

Thank you in advance.

![D28747EC-6718-429C-93F9-5BCEC45F5A73](https://user-images.githubusercontent.com/22862199/139354556-4503be71-956a-4f5f-82d4-07ee877af527.jpeg)

**Comment by OzGav** [comment, marked as answer]
Are you sure your SVG has a white background or is is actually transparent? I use a variety of devices with dark mode and I don't have the issue. It will look white in Inkscape because Inkscapes background is white...

**Reply by jhart003** [reply]
Good one! I never even thought about that.  I’ll give it a try, thank you!

**Reply by OzGav** [reply]
Cool if that the case please mark this an asnwered or come back and we will try and help further.

**Reply by jhart003** [reply]
Looks like there are several ways to do this.  The quickest seemed to be to insert this line in my SVG file:

<rect width="100%" height="100%" fill="white"/>

Thanks again for the help!
